packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -274,6 +276,11 @@ const Dispatcher: DispatcherType = {
   useResponder,
   useTransition,
   useDeferredValue,
+
+  unstable_avoidThisRender(thenable: Thenable) {
+    // this is just to satisfy flow, it should never be called.
+    // maybe I should put an invariant here
```

Comment:
Why should it never be called? It should be a no-op.

This special dispatcher is used to construct the DevTools hooks panel. DevTools executes the component function, which calls some hooks, then uses the log of hooks that were called to construct a tree of hooks. This will include any calls to `unstable_avoidThisRender`. We don't want to include `avoidThisRender` in the hook tree, which makes this kind of like `readContext`. But it also doesn't need to return anything, because it's a void function. So making it a no-op is correct.